build/dockerfiles/brew.Dockerfile

```diff
@@ -10,7 +10,7 @@
 #
 
 # https://registry.access.redhat.com/rhel8/go-toolset
-FROM rhel8/go-toolset:1.19.13-2 as builder
+FROM rhel8/go-toolset:1.20.10-3 as builder
```

Comment:
I feel like this should be restricted to 1.19 bugfixes

Reply:
yeah but some CVEs are only fixed in 1.20 and 1.21... so this might be a good thing.